### PR TITLE
Andrey Kabanets signed

### PR DESCRIPTION
**Pull request by Kabanets**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -14,14 +14,12 @@ Regardless, Stallman's opinions on the matters he is being persecuted over are n
 
 Removing RMS will hurt FSF's image and will deal a significant blow to the momentum of the free software movement. We urge you to consider your actions carefully, as what you will decide will have a serious impact on the future of the software industry.
 
-
```
Comment (nekonee) [inline index.md:17]: Please don't modify README.

File: index.md [hunk 2]
```diff
@@ -14,14 +14,12 @@ Regardless, Stallman's opinions on the matters he is being persecuted over are n
...
 **To the ambush mob who is ganging up on Richard Stallman over reasonable arguments in debate and various opinions and beliefs voiced over decades as a public figure:**
 
 You have no part in choosing the leadership of any communities. Especially not via another mob attack which does not remotely resemble a fairly conducted debate as exemplified by better people such as Richard Stallman.
 
 To sign, please [submit a pull request](https://github.com/rms-support-letter/rms-support-letter.github.io/pulls).
 
-
```
Comment (nekonee) [inline index.md:24]: Don't modify README.

**Review by nekonee — CHANGES_REQUESTED** [review]

**Comment by podvornyakva** [comment]
Please. add a file instead of edit index.md

Follow README.md instructions.

Thank you.

**Comment by imachug** [comment]
Merged manually.

**Pull request closed without merging**